src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/OverlappingTrainingDateRequest.cs

```diff
@@ -28,5 +28,9 @@ public OverlappingTrainingDateRequest(DraftApprenticeship draftApprenticeship,
             ChangeTrackingSession.TrackInsert(this);
```

Comment:
I think the event should be published from constructor
example https://github.com/SkillsFundingAgency/das-commitments/blob/master/src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs#L171

Otherwise it may get missed if the public method to emit isn't called